### ISSUE TITLE
OWASP-vulnerability in WireMock 2.25.1 JRE8 Standalone

### ISSUE THREAD

**Issue by nabeygun** [tomakehurst/wiremock#1228]
In WireMock 2.25.1 JRE8 Standalone, as defined in the below maven dependency:
```
		<dependency>
			<groupId>com.github.tomakehurst</groupId>
			<artifactId>wiremock-jre8-standalone</artifactId>
			<version>2.25.1</version>
		</dependency>
```
The following OWASP issues have been identified:
```

wiremock-jre8-standalone-2.25.1.jar: handlebars-v4.0.4.js (pkg:javascript/handlebars.js@4.0.4) : A prototype pollution vulnerability in handlebars is exploitable if an attacker can control the template

wiremock-jre8-standalone-2.25.1.jar/META-INF/maven/com.fasterxml.jackson.core/jackson-databind/pom.xml (pkg:maven/com.fasterxml.jackson.core/jackson-databind@2.9.10, cpe:2.3:a:fasterxml:jackson:2.9.10:*:*:*:*:*:*:*, cpe:2.3:a:fasterxml:jackson-databind:2.9.10:*:*:*:*:*:*:*) : CVE-2019-16942, CVE-2019-16943, CVE-2019-17531
```

It would be great to fix or update the below critical ones:
https://nvd.nist.gov/vuln/detail/CVE-2019-16942
https://nvd.nist.gov/vuln/detail/CVE-2019-16943
https://nvd.nist.gov/vuln/detail/CVE-2019-17531

Thanks

**Comment by IAmJoffa** [comment]
From what I can see from the latest config in the master branch the config for these are now, so I guess will be fixed in the next release.

```
        java8: [
            handlebars     : '4.1.2',
            jetty          : '9.4.20.v20190813',
            guava          : '27.0.1-jre',
            jackson        : '2.10.0',
            jacksonDatabind: '2.10.0',
            xmlUnit        : '2.6.2'
        ]
```

**Comment by tomakehurst** [comment]
Fixed in c4f22df64c4068a3828aac66d199c01bdabbb3c6

**Comment by tomakehurst** [comment]
The Java 8 version is the one you should be using with Java 11. The Java 7 (non-suffixed) one is EOL now so 2.27.2 is the last release.
